Project/zepeto-modules/Assets/Zepeto Gesture Module/ZepetoScript/GestureLoader.ts

```diff
@@ -17,20 +17,22 @@ export default class GestureLoader extends ZepetoScriptBehaviour {
     @SerializeField() private _prefThumb: GameObject;
 
     private _myCharacter: ZepetoCharacter;
+    private _poseIsRunning: bool;
     
     // Loop setting
     @Header("Looping Setting") 
     @Tooltip("Activate/Deactivate the looping feature") public isGestureLooping: boolean;
     @Tooltip("Waiting time in seconds before playing") @SerializeField() private _repeatInterval: number; // Waiting time in seconds before playing the gesture again.
-
+    @Tooltip("Posing time in seconds") @SerializeField() private _posingInterval: number; //Posing time in seconds 
```

Comment:
Since we're newly introducing the settings for the 'poses', where 'gestures' were the only interest, we need to categorize and rename the Settings UI.

I have two suggestions as below.

* TO-BE 1: clarify 'gesture' and 'pose' in the variable names.
* TO-BE 2: make a sub-header (if possible) for gesture and pose. (while it is clearer from the user side, we might need to use aliases(if possible) for the coder's sake. - since it is the variable name.)

| AS-IS | TO-BE 1 | TO-BE 2 |
| - | - | - |
|![as-is](https://github.com/JasperGame/zepeto-modules/assets/8598722/1c444b2f-a23d-4bcd-bead-a92e31978085)|![스크린샷 2023-08-04 오전 9 02 14](https://github.com/JasperGame/zepeto-modules/assets/8598722/07554627-4ee0-4985-b0a3-9be8752e59d6)|![스크린샷 2023-08-04 오전 9 09 43](https://github.com/JasperGame/zepeto-modules/assets/8598722/271f098c-2cd9-4b4c-8379-29ddfacc4ada)|

Which one do you think is better? or do you have 3rd option?

Reply:
Note: If the changes to the UI are to be made, the guide & screenshots should be updated accordingly 😄

Reply:
I would got for option  2 given that it's more detailed.